## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 455096`
Commit: [`b0c666abf5a102d30203683c87e8ebab9ce61c5b`](https://github.com/NixOS/nixpkgs/commit/b0c666abf5a102d30203683c87e8ebab9ce61c5b) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/b0c666abf5a102d30203683c87e8ebab9ce61c5b..pull/455096/head))
Merge: [`e80f0174ab220ade36dc08a2b9c51a7adc366852`](https://github.com/NixOS/nixpkgs/commit/e80f0174ab220ade36dc08a2b9c51a7adc366852)

Logs: https://github.com/iedame/nixpkgs-review-gha/actions/runs/18767158617


---
### `x86_64-linux`
<details>
  <summary>:x: 8 packages failed to build:</summary>
  <ul>
    <li>python312Packages.pyscf</li>
    <li>python312Packages.pyscf.dist</li>
    <li>python312Packages.simple-dftd3</li>
    <li>python312Packages.simple-dftd3.dist</li>
    <li>python313Packages.pyscf</li>
    <li>python313Packages.pyscf.dist</li>
    <li>python313Packages.simple-dftd3</li>
    <li>python313Packages.simple-dftd3.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 5 packages built:</summary>
  <ul>
    <li>cppe</li>
    <li>python312Packages.cppe</li>
    <li>python312Packages.cppe.dist</li>
    <li>python313Packages.cppe</li>
    <li>python313Packages.cppe.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>python312Packages.pyscf</summary>
<pre>  /build/source/pyscf/solvent/test/test_ddcosmo.py:304: DeprecationWarning: `scipy.special.sph_harm` is deprecated as of SciPy 1.15.0 and will be removed in SciPy 1.17.0. Please use `scipy.special.sph_harm_y` instead.
    ylm[l] = scipy.special.sph_harm(0, l, theta, varphi).real

pyscf/solvent/test/test_ddcosmo.py: 2100 warnings
  /build/source/pyscf/solvent/test/test_ddcosmo.py:306: DeprecationWarning: `scipy.special.sph_harm` is deprecated as of SciPy 1.15.0 and will be removed in SciPy 1.17.0. Please use `scipy.special.sph_harm_y` instead.
    f1 = scipy.special.sph_harm(-m, l, theta, varphi)

pyscf/solvent/test/test_ddcosmo.py: 2100 warnings
  /build/source/pyscf/solvent/test/test_ddcosmo.py:307: DeprecationWarning: `scipy.special.sph_harm` is deprecated as of SciPy 1.15.0 and will be removed in SciPy 1.17.0. Please use `scipy.special.sph_harm_y` instead.
    f2 = scipy.special.sph_harm( m, l, theta, varphi)

pyscf/tools/test/test_fcidump.py::KnownValues::test_to_scf_with_symmetry
  /build/source/pyscf/lib/misc.py:648: UserWarning: method set() only supports keyword arguments.
  Arguments &lt;pyscf.gto.mole.Mole object at 0x7fff9e0d99d0&gt; are ignored.
    warnings.warn(&#x27;method set() only supports keyword arguments.\n&#x27;

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED pyscf/solvent/test/test_pol_embed.py::TestPolEmbed::test_pe_scf_ecp - RuntimeError: Failed to converge induced dipole moments.
= 1 failed, 2654 passed, 49 skipped, 350 deselected, 200563 warnings in 4113.44s (1:08:33) =</pre>
</details>
<details>
<summary>python313Packages.pyscf</summary>
<pre>  /build/source/pyscf/solvent/test/test_ddcosmo.py:304: DeprecationWarning: `scipy.special.sph_harm` is deprecated as of SciPy 1.15.0 and will be removed in SciPy 1.17.0. Please use `scipy.special.sph_harm_y` instead.
    ylm[l] = scipy.special.sph_harm(0, l, theta, varphi).real

pyscf/solvent/test/test_ddcosmo.py: 2100 warnings
  /build/source/pyscf/solvent/test/test_ddcosmo.py:306: DeprecationWarning: `scipy.special.sph_harm` is deprecated as of SciPy 1.15.0 and will be removed in SciPy 1.17.0. Please use `scipy.special.sph_harm_y` instead.
    f1 = scipy.special.sph_harm(-m, l, theta, varphi)

pyscf/solvent/test/test_ddcosmo.py: 2100 warnings
  /build/source/pyscf/solvent/test/test_ddcosmo.py:307: DeprecationWarning: `scipy.special.sph_harm` is deprecated as of SciPy 1.15.0 and will be removed in SciPy 1.17.0. Please use `scipy.special.sph_harm_y` instead.
    f2 = scipy.special.sph_harm( m, l, theta, varphi)

pyscf/tools/test/test_fcidump.py::KnownValues::test_to_scf_with_symmetry
  /build/source/pyscf/lib/misc.py:648: UserWarning: method set() only supports keyword arguments.
  Arguments &lt;pyscf.gto.mole.Mole object at 0x7fffc0938190&gt; are ignored.
    warnings.warn(&#x27;method set() only supports keyword arguments.\n&#x27;

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED pyscf/solvent/test/test_pol_embed.py::TestPolEmbed::test_pe_scf_ecp - RuntimeError: Failed to converge induced dipole moments.
= 1 failed, 2654 passed, 49 skipped, 350 deselected, 200563 warnings in 4109.14s (1:08:29) =</pre>
</details>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 5 packages built:</summary>
  <ul>
    <li>cppe</li>
    <li>python312Packages.cppe</li>
    <li>python312Packages.cppe.dist</li>
    <li>python313Packages.cppe</li>
    <li>python313Packages.cppe.dist</li>
  </ul>
</details>

---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python312Packages.pyscf</li>
    <li>python312Packages.pyscf.dist</li>
    <li>python313Packages.pyscf</li>
    <li>python313Packages.pyscf.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 5 packages built:</summary>
  <ul>
    <li>cppe</li>
    <li>python312Packages.cppe</li>
    <li>python312Packages.cppe.dist</li>
    <li>python313Packages.cppe</li>
    <li>python313Packages.cppe.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>python312Packages.pyscf</summary>
<pre>
pyscf/solvent/test/test_ddcosmo.py: 2100 warnings
  /nix/build/nix-10920-1391378648/source/pyscf/solvent/test/test_ddcosmo.py:306: DeprecationWarning: `scipy.special.sph_harm` is deprecated as of SciPy 1.15.0 and will be removed in SciPy 1.17.0. Please use `scipy.special.sph_harm_y` instead.
    f1 = scipy.special.sph_harm(-m, l, theta, varphi)

pyscf/solvent/test/test_ddcosmo.py: 2100 warnings
  /nix/build/nix-10920-1391378648/source/pyscf/solvent/test/test_ddcosmo.py:307: DeprecationWarning: `scipy.special.sph_harm` is deprecated as of SciPy 1.15.0 and will be removed in SciPy 1.17.0. Please use `scipy.special.sph_harm_y` instead.
    f2 = scipy.special.sph_harm( m, l, theta, varphi)

pyscf/tools/test/test_fcidump.py::KnownValues::test_to_scf_with_symmetry
  /nix/build/nix-10920-1391378648/source/pyscf/lib/misc.py:648: UserWarning: method set() only supports keyword arguments.
  Arguments &lt;pyscf.gto.mole.Mole object at 0x15a1ec950&gt; are ignored.
    warnings.warn(&#x27;method set() only supports keyword arguments.\n&#x27;

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED pyscf/df/test/test_df_hessian.py::KnownValues::test_uks_hess - AssertionError: np.float64(0.0005703997261376492) != 0 within 4 places (np....
FAILED pyscf/mcscf/test/test_umc1step.py::KnownValues::test_ucasscf - AssertionError: np.float64(-75.73025017306321) != -75.7460662487894 within ...
FAILED pyscf/solvent/test/test_pol_embed.py::TestPolEmbed::test_pe_scf_ecp - RuntimeError: Failed to converge induced dipole moments.
= 3 failed, 2652 passed, 49 skipped, 350 deselected, 199921 warnings in 5401.46s (1:30:01) =</pre>
</details>
<details>
<summary>python313Packages.pyscf</summary>
<pre>    ylm[l] = scipy.special.sph_harm(0, l, theta, varphi).real

pyscf/solvent/test/test_ddcosmo.py: 2100 warnings
  /nix/build/nix-10920-1391378647/source/pyscf/solvent/test/test_ddcosmo.py:306: DeprecationWarning: `scipy.special.sph_harm` is deprecated as of SciPy 1.15.0 and will be removed in SciPy 1.17.0. Please use `scipy.special.sph_harm_y` instead.
    f1 = scipy.special.sph_harm(-m, l, theta, varphi)

pyscf/solvent/test/test_ddcosmo.py: 2100 warnings
  /nix/build/nix-10920-1391378647/source/pyscf/solvent/test/test_ddcosmo.py:307: DeprecationWarning: `scipy.special.sph_harm` is deprecated as of SciPy 1.15.0 and will be removed in SciPy 1.17.0. Please use `scipy.special.sph_harm_y` instead.
    f2 = scipy.special.sph_harm( m, l, theta, varphi)

pyscf/tools/test/test_fcidump.py::KnownValues::test_to_scf_with_symmetry
  /nix/build/nix-10920-1391378647/source/pyscf/lib/misc.py:648: UserWarning: method set() only supports keyword arguments.
  Arguments &lt;pyscf.gto.mole.Mole object at 0x158a802d0&gt; are ignored.
    warnings.warn(&#x27;method set() only supports keyword arguments.\n&#x27;

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED pyscf/mcscf/test/test_umc1step.py::KnownValues::test_ucasscf - AssertionError: np.float64(-75.73025017306321) != -75.7460662487894 within ...
FAILED pyscf/solvent/test/test_pol_embed.py::TestPolEmbed::test_pe_scf_ecp - RuntimeError: Failed to converge induced dipole moments.
= 2 failed, 2653 passed, 49 skipped, 350 deselected, 199921 warnings in 5411.05s (1:30:11) =</pre>
</details>
</details>

---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python312Packages.pyscf</li>
    <li>python312Packages.pyscf.dist</li>
    <li>python313Packages.pyscf</li>
    <li>python313Packages.pyscf.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 5 packages built:</summary>
  <ul>
    <li>cppe</li>
    <li>python312Packages.cppe</li>
    <li>python312Packages.cppe.dist</li>
    <li>python313Packages.cppe</li>
    <li>python313Packages.cppe.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>python312Packages.pyscf</summary>
<pre>  /nix/build/nix-7985-3709674225/source/pyscf/solvent/test/test_ddcosmo.py:304: DeprecationWarning: `scipy.special.sph_harm` is deprecated as of SciPy 1.15.0 and will be removed in SciPy 1.17.0. Please use `scipy.special.sph_harm_y` instead.
    ylm[l] = scipy.special.sph_harm(0, l, theta, varphi).real

pyscf/solvent/test/test_ddcosmo.py: 2100 warnings
  /nix/build/nix-7985-3709674225/source/pyscf/solvent/test/test_ddcosmo.py:306: DeprecationWarning: `scipy.special.sph_harm` is deprecated as of SciPy 1.15.0 and will be removed in SciPy 1.17.0. Please use `scipy.special.sph_harm_y` instead.
    f1 = scipy.special.sph_harm(-m, l, theta, varphi)

pyscf/solvent/test/test_ddcosmo.py: 2100 warnings
  /nix/build/nix-7985-3709674225/source/pyscf/solvent/test/test_ddcosmo.py:307: DeprecationWarning: `scipy.special.sph_harm` is deprecated as of SciPy 1.15.0 and will be removed in SciPy 1.17.0. Please use `scipy.special.sph_harm_y` instead.
    f2 = scipy.special.sph_harm( m, l, theta, varphi)

pyscf/tools/test/test_fcidump.py::KnownValues::test_to_scf_with_symmetry
  /nix/build/nix-7985-3709674225/source/pyscf/lib/misc.py:648: UserWarning: method set() only supports keyword arguments.
  Arguments &lt;pyscf.gto.mole.Mole object at 0x122922c00&gt; are ignored.
    warnings.warn(&#x27;method set() only supports keyword arguments.\n&#x27;

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED pyscf/solvent/test/test_pol_embed.py::TestPolEmbed::test_pe_scf_ecp - RuntimeError: Failed to converge induced dipole moments.
= 1 failed, 2654 passed, 49 skipped, 350 deselected, 200613 warnings in 3683.49s (1:01:23) =</pre>
</details>
<details>
<summary>python313Packages.pyscf</summary>
<pre>  /nix/build/nix-7985-3709674226/source/pyscf/solvent/test/test_ddcosmo.py:304: DeprecationWarning: `scipy.special.sph_harm` is deprecated as of SciPy 1.15.0 and will be removed in SciPy 1.17.0. Please use `scipy.special.sph_harm_y` instead.
    ylm[l] = scipy.special.sph_harm(0, l, theta, varphi).real

pyscf/solvent/test/test_ddcosmo.py: 2100 warnings
  /nix/build/nix-7985-3709674226/source/pyscf/solvent/test/test_ddcosmo.py:306: DeprecationWarning: `scipy.special.sph_harm` is deprecated as of SciPy 1.15.0 and will be removed in SciPy 1.17.0. Please use `scipy.special.sph_harm_y` instead.
    f1 = scipy.special.sph_harm(-m, l, theta, varphi)

pyscf/solvent/test/test_ddcosmo.py: 2100 warnings
  /nix/build/nix-7985-3709674226/source/pyscf/solvent/test/test_ddcosmo.py:307: DeprecationWarning: `scipy.special.sph_harm` is deprecated as of SciPy 1.15.0 and will be removed in SciPy 1.17.0. Please use `scipy.special.sph_harm_y` instead.
    f2 = scipy.special.sph_harm( m, l, theta, varphi)

pyscf/tools/test/test_fcidump.py::KnownValues::test_to_scf_with_symmetry
  /nix/build/nix-7985-3709674226/source/pyscf/lib/misc.py:648: UserWarning: method set() only supports keyword arguments.
  Arguments &lt;pyscf.gto.mole.Mole object at 0x11636c410&gt; are ignored.
    warnings.warn(&#x27;method set() only supports keyword arguments.\n&#x27;

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED pyscf/solvent/test/test_pol_embed.py::TestPolEmbed::test_pe_scf_ecp - RuntimeError: Failed to converge induced dipole moments.
= 1 failed, 2654 passed, 49 skipped, 350 deselected, 200613 warnings in 3678.92s (1:01:18) =</pre>
</details>
</details>

--- a/pkgs/by-name/cp/cppe/cmake-minimum-required.patch
+++ b/pkgs/by-name/cp/cppe/cmake-minimum-required.patch
@@ -1,0 +1,22 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 5d5d359..ba8992c 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -1,5 +1,5 @@
+-cmake_minimum_required(VERSION 3.0.0)
+-cmake_policy(VERSION 3.0.0)
++cmake_minimum_required(VERSION 3.10)
++cmake_policy(VERSION 3.10)
+ 
+ project(cppe LANGUAGES CXX)
+ 
+diff --git a/cppe/CMakeLists.txt b/cppe/CMakeLists.txt
+index 6780f18..17ac4b9 100644
+--- a/cppe/CMakeLists.txt
++++ b/cppe/CMakeLists.txt
+@@ -1,4 +1,4 @@
+-cmake_minimum_required(VERSION 3.0.0)
++cmake_minimum_required(VERSION 3.10)
+ project(cppe-core VERSION 0.3.1 LANGUAGES CXX)
+ 
+ include(GNUInstallDirs)

--- a/pkgs/by-name/cp/cppe/package.nix
+++ b/pkgs/by-name/cp/cppe/package.nix
@@ -17,6 +17,10 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-guM7+ZWDJLcAUJtPkKLvC4LYSA2eBvER7cgwPZ7FxHw=";
   };
 
+  patches = [
+    ./cmake-minimum-required.patch
+  ];
+
   nativeBuildInputs = [ cmake ] ++ lib.optionals stdenv.cc.isClang [ llvmPackages.openmp ];
 
   cmakeFlags = [ "-DCMAKE_INSTALL_LIBDIR=lib" ];

